docs/eventing/sink/kafka-sink.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Apache Kafka Sink"
+weight: 30
+type: "docs"
+---
+
+This page shows how to install and configure Apache Kafka Sink.
+
+## Prerequisites
+
+[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component).
+
+## Installation
+
+1. Install the Kafka controller:
+
+    ```bash
+    kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-controller.yaml" >}}
+    ```
+
+1. Install the Kafka Sink data plane:
+
+    ```bash
+    kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-sink.yaml" >}}
+    ```
+
+1. Verify that `kafka-controller` and `kafka-sink-receiver` are running:
```

Comment:
```suggestion
1. Verify that `kafka-controller` and `kafka-sink-receiver` deployments are running:
```

Not sure if that's what these are, but I'd specify if they're deployments, services, pods, controllers, etc because it's not clear.